--- buildit	2013-12-08 16:36:25.552841502 +0100
+++ buildit	2013-12-08 16:37:01.648843150 +0100
@@ -27,7 +27,7 @@
   RC_ProjectSourceVersion=1
 fi
 
-EXTRA_FLAGS="-std=c++0x -stdlib=libc++ -fstrict-aliasing -Wstrict-aliasing=2 \
+EXTRA_FLAGS="-std=c++0x `cat ../../libcxx-conf` -fstrict-aliasing -Wstrict-aliasing=2 \
              -Wsign-conversion -Wshadow -Wconversion -Wunused-variable \
              -Wmissing-field-initializers -Wchar-subscripts -Wmismatched-tags \
              -Wmissing-braces -Wshorten-64-to-32 -Wsign-compare \
@@ -92,6 +92,7 @@
   ;;
 esac
 $CC *.o $RC_CFLAGS $LDSHARED_FLAGS $EXTRA_FLAGS
+ar rs libc++abi.a *.o && ln -sf libc++abi.so* libc++abi.so
 
 if [ -z $RC_XBS ]
 then
